### PR TITLE
V100.2.0

### PR DESCRIPTION
**Pull request by zinfin**
Hi @gunt0001 and @hansonchris, This PR updates the Android runtime SDK to 100.2.0.  Would you mind reviewing?

### CODE REVIEW

**Comment by gunt0001** [comment]
@alan-edi - please review.

File: emu-app/src/main/java/com/esri/android/ecologicalmarineunitexplorer/data/DataManager.java [hunk 1]
```diff
@@ -271,31 +275,36 @@ public void queryEmuByDepth (final Integer depth, final ServiceApi.EMUByDepthCal
...
           } catch (final Exception e) {
-            Log.e("DataManager", "Error querying EMU by depth " + e.getMessage());
+            Log.e(TAG, "Error querying EMU by depth " + e.getMessage());
             callback.onPolygonsRetrieved(null);
           }
         }
       });
 
+
+
```
Comment (alan-edi) [inline emu-app/src/main/java/com/esri/android/ecologicalmarineunitexplorer/data/DataManager.java]: Are these extra blank lines deliberate?

File: emu-app/src/main/java/com/esri/android/ecologicalmarineunitexplorer/map/MapPresenter.java [hunk 2]
```diff
@@ -220,7 +218,7 @@ private void cacheInitialDepthLayer(){
       @Override public void onPolygonsRetrieved(final FeatureLayer layer) {
         if (layer != null){
           Log.i("MapPresenter", "Initial depth layer downloaded");
-          layer.setDefinitionExpression(" Depth = 0");
+         // layer.setDefinitionExpression(" Depth = 0");
```
Comment (alan-edi) [inline emu-app/src/main/java/com/esri/android/ecologicalmarineunitexplorer/map/MapPresenter.java]: Is it deliberate to leave in this commented out line?

**Comment by zinfin** [comment]
@alan-edi Thanks!

**Pull request merged**